python/build-wheel-manylinux1.sh

```diff
@@ -35,6 +35,18 @@ export PATH=$PATH:/root/bin
 rm -f /usr/bin/python2
 ln -s /opt/python/cp27-cp27m/bin/python2 /usr/bin/python2
 
+# Install and use the latest version of Node.js in order to build the dashboard.
+curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.34.0/install.sh | bash
+source $HOME/.nvm/nvm.sh
+nvm install node
+nvm use node
+
+# Build the dashboard so its static assets can be included in the wheel.
+pushd python/ray/dashboard/client
+  npm ci
+  npm run build
+popd
```

Comment:
This block may need to be done inside of the for loop because the results may get removed by the `git clean -f -f -x -d -e .whl` in the for loop. If they don't get removed then it's probably because they are protected by a gitignore file, but probably safer to just redo it.

Reply:
It looks like `git clean -x` causes even ignored files (such the dashboard build files) to be removed. I've revised the logic in `build-wheel-macos.sh` and `build-wheel-manylinux1.sh` to rebuild the dashboard inside the `for` loop.